vnext/Microsoft.ReactNative/QuirkSettings.idl

```diff
@@ -28,10 +34,10 @@ namespace Microsoft.ReactNative
     static void SetAcceptSelfSigned(ReactInstanceSettings settings, Boolean value);
 
     DOC_STRING(
-      "Runtime setting that can be used to skip the default back handler logic.  "
-      "If you have an application that wants to handle back bahavior in native code "
-      "instead of JavaScript, setting this to false will prevent the native module "
-      "from handling the events. Default: true")
-    static void SetEnableBackHandler(ReactInstanceSettings settings, Boolean value);
+      "By default `react-native-windows` will handle various back events and forward them to JavaScript. "
+      "Setting this to `BackNavigationHandlerKind.Native` prevents `react-native-windows` from handling "
+      "these events.  This will allow applications to handle back navigation in native code, but will prevent "
```

Comment:
sorry for another suggestion:
maybe:
prevents `react-native-windows` form handling these events, including forwarding to JavaScript.

Reply:
Note now it clicked what the original wording was. I got confused by the term 'native code' We were talking about 2 types of native code: a) Our native code in rnw that does the forwarding and b) customer native code.